azure-pipeline-public-nuget-publish.yml

```diff
@@ -6,17 +6,13 @@ trigger:
 pr: none
 
 pool:
-  vmImage: 'macOS-13'
+  vmImage: 'macOS-15'
 
 variables:
   - group: nuget.org
 
 steps:
 
-- bash: |
-    sudo xcode-select -s /Applications/Xcode_15.2.app/Contents/Developer
```

Comment:
introduce xcode version file and use xcodes to pint the xcode version to 16.0 explicitly

Reply:
updated